docs_src/parameter_types/enum/tutorial003.py

```diff
@@ -0,0 +1,18 @@
+import enum
+import logging
+
+import typer
+
+
+class LogLevel(enum.Enum):
+    debug = logging.DEBUG
+    info = logging.INFO
+    warning = logging.WARNING
+
+
+def main(log_level: LogLevel = typer.Option(LogLevel.warning, names=True)):
```

Comment:
`LogLevel.warning` didn't work as the default value when I tried it.

```console
$ python docs_src/parameter_types/enum/tutorial003.py
Usage: tutorial003.py [OPTIONS]
Try 'tutorial003.py --help' for help.

Error: Invalid value for '--log-level': invalid choice: LogLevel.warning. (choose from debug, info, warning)
```

Can you please fix this and add a test case that runs `runner.invoke(app)` to exercise the default value?

Reply:
Good catch! I pushed a fix for this.